requirements.txt

```diff
@@ -1,13 +0,0 @@
-black; implementation_name == "cpython"
-cmake
-codecov
-coverage
-flake8
-jinja2
-lxml
-pygments == 2.7.4
-pytest
-pytest-cov
-twine
-wheel
-yaxmldiff
```

Comment:
I'd have a preference for keeping the requirements.txt file as it is machine-readable and may assist with other tools. E.g. I install library dependencies (jinja2, lxml, pygments, pytest, yaxmldiff) in a development virtualenv so that my editor can provide autocomplete. But all the tools (black, cmake, codecov, coverage, flake8, pytest-cov, twine, wheel) can be managed via Nox.

Reply:
Ok. Should the dependencies in nox be installed via the requirements.txt?

Reply:
Hmm, on second thought jinja2, lxml, and pygments will already be pulled in by `pip install -e .` which is part of the developer setup instructions, and also part of the Nox session doing testing. So my argument would only affect pytest and yaxmldiff, the latter of which is practically irrelevant.

In that case, leaving everything as it currently is might be OK.